src/phpdoc/parser.go

```diff
@@ -7,6 +7,12 @@ import (
 	"github.com/z7zmey/php-parser/pkg/token"
 )
 
+type PhpDoc struct {
+	Raw        string
+	Parsed     []CommentPart
+	Suspicious bool
```

Comment:
As I see, we use this field in 2 unconnected places only once per comment.
Would it be hard to get rid of this field and check whether a comment is suspicious inside a linter code?
I feel like checking whether something is suspicious is a linter job, not the IR converter job.
We do mark strings as invalid as it would otherwise be impossible to report such errors, but this case doesn't look like exceptional enough.